atp/client.go

```diff
@@ -465,28 +520,34 @@ func (c *client) prepareResultChannels(
 	return nil
 }
 
-// getResultV2 works with the channels that communicate with the RuntimeMessage loop.
+// getResultV2 communicates with the RuntimeMessage loop to get the .
```

Comment:
We lost a word at the end of the sentence.

Reply:
Fixed.